src/System.Runtime.InteropServices/src/System.Runtime.InteropServices.csproj

```diff
@@ -22,6 +22,7 @@
     <Compile Include="System\Runtime\CompilerServices\Attributes.cs" />
     <Compile Include="System\Runtime\InteropServices\Attributes.cs" />
     <Compile Include="System\Runtime\InteropServices\ComAwareEventInfo.cs" />
+    <Compile Include="System\Runtime\InteropServices\ComEventInterfaceAttribute.cs" Condition="'$(TargetGroup)' != 'uapaot'" />
```

Comment:
The right fix is to delete it in System.Private.Interop and keep it here. We want to have the implementation of a given type in the same place (ie either in corefx or in the runtime specific repo). corefx for one runtime and runtime specific repo for other runtime is bad.

Note that ComAwareEventInfo has the same problem - it is exists in both System.Private.Interop and here. It should be deleted from System.Private.Interop as well.